compiler-rt/lib/scudo/standalone/include/scudo/interface.h

```diff
@@ -123,6 +123,8 @@ size_t __scudo_get_region_info_size(void);
 const char *__scudo_get_ring_buffer_addr(void);
 size_t __scudo_get_ring_buffer_size(void);
 
+bool __scudo_resize_ring_buffer(int);
+
```

Comment:
I barely remember that we had discussion over the email about how we will implement this. I think it'll be set through mallopt, right? Besides, I'm not sure if we have agreed to implement it like this. For example, can we just grow the buffer when it meets certain condition? Or can't we use the env variable to set the size before the start of debugging app?

Reply:
Yes, this will be used in the mallopt, but we need to do additional stuff in libc (see the AOSP draft CL linked in description)